pkg/deploy/generator/scripts/util-services.sh

```diff
@@ -303,6 +303,199 @@ WantedBy=multi-user.target"
     write_file aro_portal_service_filename aro_portal_service_file true
 }
 
+# configure_service_aro_mise
+# args:
+# 1) image - nameref, string; MISE container image
+# 2) network - nameref, string; podman network name to be attached
+configure_service_aro_mise() {
+    local -n image="$1"
+    local -n network="$2"
+    log "starting"
+    log "Configuring aro-mise service"
+
+    # shellcheck disable=SC2034
+    local -r aro_mise_service_conf_filename='/etc/sysconfig/aro-mise'
+    # shellcheck disable=SC2034
+    local -r aro_mise_service_conf_file="FPCLIENTID='$FPCLIENTID'
+FPTENANTID='$FPTENANTID'
+MISEIMAGE='$image'
+MISEVALIDAUDIENCES='$MISEVALIDAUDIENCES'
+MISEVALIDAPPIDS='$MISEVALIDAPPIDS'
+LOGININSTANCE='$LOGININSTANCE'"
+
+    write_file aro_mise_service_conf_filename aro_mise_service_conf_file true
+
+    mkdir -p /app/mise
+    cat >/app/mise/appsettings.json <<EOF
```

Comment:
Make use of `write_file`. You should be able to use single quotes `''` to allow for using double quotes `""` within the variable that you pass to `write_file`.  
Worst case, you can always escape them with `\"`, but you shouldn't need to do that.